evap/staff/importers/person.py

```diff
@@ -0,0 +1,100 @@
+from typing import Iterable
+
+from django.utils.html import format_html
+from django.utils.translation import gettext as _
+
+from evap.evaluation.models import Contribution, Evaluation, UserProfile
+from evap.staff.tools import ImportType, create_user_list_html_string_for_message
+
+from .base import ImporterLog
+from .user import import_users
```

Comment:
I used relative imports in `staff/importers/` so the whole directly can be moved without having to adjust anything, because I considered the file structure an implementation detail not relevant to the rest of the code. Absolute imports should also work fine.

Do you have a good reason for using absolute imports? The absolute path is longer, so I don't really see a benefit.

Reply:
One reason for me would be that when I have a `UserImporter` and a method that returns `ImporterLog`, I could not see what to import after going to the `user.py` file. But your point makes sense too, so you can decide